### DISCUSSION TITLE
Path Variables

### DISCUSSION THREAD

**Discussion by wittling** [postmanlabs/postman-flows#65] [Q&A] [answered]
This is a cool new feature (Flows), and I realize it's early beta. So maybe what I am trying to do isn't ready yet or doesn't work yet. But here is what I am trying to do:

1. Get a list of VM Instances (works fine)
2. Loop through, looking for a status of failed (works fine - there is no API query to get instances w parm = status)
3. Save list of VM interfaces as variable (works fine)
4. Loop through the list of interfaces, saving the network id. (works fine)
5. With the Network Id in hand as variable, make a call to Get Networks. But this call requires path variable ":id" **_and this does NOT work._**
6. Make call to GetNetworks, and use the ForEach to loop through and validate, choosing the one with variable network id, and this does not work because Validate looks like it only takes a typed-in constant as comparator and not a previously saved variable in bracket format "{{var}}" or "$var".

So right now, not having a way to use a Path Variable dynamically, or use a variable in Validate, I am kind of dead in the water as far as trying to achieve the end goal (which is to get the network info so I can delete the IP of a failed instance from DHCP server pool).

![Screen Shot 2021-10-25 at 11 05 14 AM](https://user-images.githubusercontent.com/31422508/138721393-738ef73b-7753-4df8-86b5-5fdd1c6c5af6.png)

**Comment by samay-v** [comment, marked as answer]
Hey @wittling 
Have you referenced the `networkid` variable in the request as the value of your path-variable? This is needed in order to pass the variable data. 
It would look something like this:

![image](https://user-images.githubusercontent.com/28357106/138817298-65f229e9-a435-4154-b6a0-1d4d64686142.png)
